Morphic.Client/DefaultPreferences.json

```diff
@@ -23,6 +23,10 @@
       {
         "type": "control",
         "feature": "contrast"
+      },
+      {
+        "type": "control",
+        "feature": "speech"
```

Comment:
Gotta figure out how we want to add this to the list.  I think it'll likely replace the narrator item.

We probably need a way to get new buttons on existing installs because those have saved the default set as their own preferences and won't see any update to the defaults.

For this one we might be able to cheat and just redefine the `reader` handler to be select-to-speak instead of narrator.

Reply:
Talked with Chris and we decided in this case to simply change the behavior of the `reader` item from narrator to this select-to-speak functionality.

Reply:
done